### PR TITLE
Add in a line in the tutorial page help text on how to reorder the cards in the tutorial

### PR DESCRIPTION
**Pull request by mschwamb**
There was no help text on how to reorder tutorial. Only if you move your move over the gray bit does the mouse icon change to clue you in. So I added in a line into the help text on the tutorial page.

### CODE REVIEW

File: app/pages/lab/tutorial.cjsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ module.exports = React.createClass
         <p>This is the place to give the volunteers a preview of the data they'll be working with and of the steps they'll be taking to make classifications. It's also a good place to mention any common "gotchas" users might face.</p>
         <p>However, it's also important to keep this as short as possible so volunteers can get started classifying as soon as possible!</p>
         <p><strong>Tutorials should be linked to a workflow on its workflow editor page to pop up for your project.</strong></p>
+	<p> <strong> You can reorder the stepss in your tutorial be clicking and dragging on the left gray tab </strong> </p>
```
Comment (srallen) [inline app/pages/lab/tutorial.cjsx]: Looks like there are a couple of typos here: `steps` instead of `stepss` and `by clicking` instead of `be clicking`

**Comment by mschwamb** [comment]
gah! sorry about that @srallen I looked at it a bunch of times and missed that. I'm really sorry about that. I've fixed the typos.

**Comment by srallen** [comment]
@mschwamb, no worries. One other change is that the indentation of the new paragraph needs to match the others. Coffeescript is white space significant and the javascript won't compile if the indents are off. Looks like it should be 4 indents (or 8 spaces).

**Comment by mschwamb** [comment]
Sorry @srallen I thought I used spaces, but I used a tab. Sorry! It should be fixed now.

**Pull request merged**